.github/workflows/cypress.yml

```diff
@@ -53,40 +75,23 @@ jobs:
           path: ${{ steps.composer-cache.outputs.dir }}
           key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
           restore-keys: |
-            ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-
-      - name: Setup Node
-        uses: actions/setup-node@v2
-        with:
-          node-version: 14
+            ${{ runner.os }}-composer-
 
-      - name: Cache node modules
-        uses: actions/cache@v2
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-
-      - name: Log debug information
+      - name: Show versions
         run: |
           php --version
           composer --version
           node --version
           npm --version
 
-      - name: Validate composer.json and composer.lock files
+      - name: Validate composer.json and composer.lock
         run: composer validate
 
       - name: Install PHP Dependencies
-        run: composer install --no-progress --no-dev --optimize-autoloader
+        run: composer install --no-progress --no-dev --optimize-autoloader --prefer-dist
 
-      - name: Run NPM Install
-        run: npm install
+      - name: NPM Install
+        run: npm install --legacy-peer-deps
```

Comment:
We may not have to have the `--legacy-peer-deps` flag. If you can run a normal `npm install` without getting any errors/warnings, we don't need it. If you get a notice about missing peer dependencies, then we either need to add those or add this flag. It was easier to add the flag on the Bluehost plugin, but may not be required on the other plugins.